queries.sql

```diff
@@ -16,3 +16,35 @@ SELECT * from animals WHERE name != 'Gabumon';
 
 SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
 
+-- day 2
+
+BEGIN;
+UPDATE animals SET species = 'unspecified';
+ROLLBACK;
```

Comment:
- [ ] Kindly make the changes asked by the previous reviewer. He made it very clear what is supposed to be done  [here](https://github.com/OpondoG/Vet-Clinic-Database/pull/2/files#r1050716243). But still if you don't understand you can simply add  `SELECT species from animals;` before and after `Rollback`.